static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,189 @@
+    .filter(project => project && !project.firstEvent) as Project[];
+  return (
+    <AnimatePresence>
+      {projects.length > 0 && <Heading>{t('The Basics')}</Heading>}
+      {projects.length > 0 && (
+        <TaskCard key="onboarding-continue-card">
+          <Title>{t('Project to Setup')}</Title>
+          <OnboardingTaskProjectList>
+            {projects.slice(0, MAX_PROJECT_COUNT).map(p => (
+              <OnboardingTaskProjectListItem
+                key={p.id}
+                to={`/onboarding/${org.slug}/setup-docs/?project_id=${p.id}`}
+                onClick={() => {
+                  trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                    platform: p.platform,
+                    organization: org,
+                  });
+                }}
+              >
+                <OnboardingTaskProjectListItemInner>
+                  <StyledPlatformIcon platform={p.platform || 'default'} />
+                  {p.slug}
+                  <PulsingIndicator />
+                  <PulsingIndicatorText>{t('Waiting for event')}</PulsingIndicatorText>
+                  <IconChevron direction="right" />
+                </OnboardingTaskProjectListItemInner>
+              </OnboardingTaskProjectListItem>
+            ))}
+            {projects.length > MAX_PROJECT_COUNT && (
+              <OnboardingTaskProjectListItem
+                to={`/onboarding/${org.slug}/setup-docs/`}
+                onClick={() => {
+                  trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                    organization: org,
+                  });
+                }}
+              >
+                <OnboardingTaskProjectListItemInner>
+                  <StyledAndMoreIcon />
+                  {tct('and [num] more', {num: projects.length - MAX_PROJECT_COUNT})}
+                </OnboardingTaskProjectListItemInner>
+              </OnboardingTaskProjectListItem>
+            )}
+          </OnboardingTaskProjectList>
+          <SkipConfirm onSkip={handleSkip}>
+            {({skip}) => <StyledIconClose size="xs" onClick={skip} />}
+          </SkipConfirm>
+        </TaskCard>
+      )}
+    </AnimatePresence>
+  );
+}
+
+const Heading = styled(motion.div)`
+  display: flex;
+  color: ${p => p.theme.purple300};
+  font-size: ${p => p.theme.fontSizeExtraSmall};
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 1;
+  margin-top: ${space(3)};
+`;
+Heading.defaultProps = {
+  exit: 'exit',
+  layout: true,
+  variants: {
+    exit: {
+      height: 0,
+      marginTop: 0,
+      opacity: 0,
+      transition: {duration: 0.3},
+    },
+  },
+};
+
+const Title = styled('div')`
+  font-weight: 600;
+  margin: ${space(2)} ${space(3)} 0;
+`;
+
+const TaskCard = styled(motion(Card))`
+  position: relative;
+  overflow: visible;
+  display: flex;
+  flex-direction: column;
+`;
+TaskCard.defaultProps = {
+  exit: 'exit',
+  layout: true,
```

Comment:
Is the `layout` prop actually needed? Are we doing a layout transition between elements? Any chance you could post a video of the animations?

Reply:
Ok yeah I guess `layout` is not necessary here. Thanks for pointing that out!
I've posted the video in the PR description.